### ISSUE TITLE
[Suggestion] Unique spider behavious

### ISSUE THREAD

**Issue by theishiopian** [TheCyberBrick/Spiders-2.0#1]
I wasn't sure if you are accepting suggestions, so if you aren't go ahead and close this. Otherwise, I have some ideas for behaviors that spiders could possibly posses in addition to their improved pathfinding:

- spiders can "swim" in cobwebs similar to how drowned swim in water.
- spiders will shoot webs at the player if the player is faster than them. this takes the form of a projectile that encases the player in webs on impact, or just places a web block on impact with blocks. Webs will decay if doMobGreifing is set to false.
- spiders now sometimes spawn with "spider nests" in overworld forests, with cocoon blocks providing loot. these nests are similar to the ones in mineshafts with cave spiders and contain a central spawner.
- if a spider can't reach the player, but the player is in web range, the spider will shoot webs repeatedly until a bridge of webbing is formed.

- cave spiders will attempt to crawl on ceilings and drop on top of the player.

**Comment by theishiopian** [comment]
![2020-10-01_17 53 37](https://user-images.githubusercontent.com/8585975/94877495-86f29f00-040f-11eb-848e-60d877d6fae5.png)
 A spider nest

**Comment by TheCyberBrick** [comment]
Yeah, I'm not sure yet whether I want to keep the mod as simple as it is now or actually add new significant features like this.

**Comment by theishiopian** [comment]
Thats fair.
